src/apps/Altinn.Authorization/src/Altinn.Authorization/Services/Implementation/ResourceRegistryWrapper.cs

```diff
@@ -114,6 +116,26 @@ public async Task<IEnumerable<AccessListResourceMembershipWithActionFilterDto>>
             return memberships;
         }
 
+        /// <inheritdoc/>
+        public async Task<IEnumerable<AccessListInfoDto>> GetMembershipsForParty(PartyUrn partyUrn, CancellationToken cancellationToken = default)
+        {
+            string cacheKey = $"AccListMemb|{partyUrn}";
```

Comment:
PartyUrn modellen direkte i string interpolation gir string verdi av urn ikke objekt id antar jeg?
Alternativt skulle det evt. vært bare selve uuid verdien

Reply:
Det funker, men har endret til partyuuid typen eksplisitt for å få feil i klient hvis man prøver noe annet